pyproject.toml

```diff
@@ -14,7 +14,7 @@ uvicorn = { extras = ["standard"], version = "^0.21.1" }
 sqlalchemy = { extras = ["asyncio"], version = "^2.0.11" }
 asyncpg = "^0.27.0"
 alembic = "^1.10.4"
-httpx = "^0.24.0"
+fastapi-users = {extras = ["oauth", "sqlalchemy"], version = "^11.0.0"}
```

Comment:
Are we switching from `httpx` to `fastapi-users`? Do you mind explaining the `extras` part? Do these libraries all link together nicely?

Reply:
I removed the explicit httpx version here because it was conflicting with the requirements of other dependencies. We still have httpx in the project, but it's brought in by a dependency rather than us directly including it. `httpx` is like the popular requests package, but it supports async/await.

[FastAPI-Users](https://fastapi-users.github.io/fastapi-users/11.0/) lets us get a tested authentication library essentially for free. Extras are kind if like optional sub-packages within a package. Since we're using SQLAlchemy as our ORM, we bring that extra in. There's an alternative for beanie, the NoSql package. We also bring in OAuth for sign in with google

Reply:
Thanks for explaining :) Sounds good.